modules/nf-core/custom/matrixfilter/meta.yml

```diff
@@ -42,29 +42,38 @@ output:
           description: |
             Groovy Map containing information on experiment.
             e.g. [ id:'test' ]
+          pattern: "*.filtered.tsv"
       - "*.filtered.tsv":
-          type: file
-          description: Filtered version of input matrix
+          type: map
+          description: |
+            Groovy Map containing information on experiment.
+            e.g. [ id:'test' ]
           pattern: "*.filtered.tsv"
   - tests:
       - meta:
           type: map
           description: |
             Groovy Map containing information on experiment.
             e.g. [ id:'test' ]
+          pattern: "*.tests.tsv"
       - "*.tests.tsv":
-          type: file
-          description: Boolean matrix with pass/ fail status for each test on each feature
+          type: map
+          description: |
+            Groovy Map containing information on experiment.
+            e.g. [ id:'test' ]
           pattern: "*.tests.tsv"
   - session_info:
       - meta:
           type: map
           description: |
             Groovy Map containing information on experiment.
             e.g. [ id:'test' ]
-      - R_sessionInfo.log:
-          type: file
-          description: Log file containing R session information
+          pattern: "*.log"
+      - "*R_sessionInfo.log":
```

Comment:
I guess this line is the only line that would need to be changed but not the rest.

Reply:
@nschcolnicov you can revert all the other changes to the meta.yml :)

Reply:
I wrote you a message on slack for the bug report :)